vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -67,6 +68,8 @@ def dash_ag_grid(data_frame, **kwargs):
                 "buttons": ["apply", "reset"],
                 "closeOnApply": True,
             },
+            "flex": 1,
+            "minWidth": 70,
```

Comment:
Can this also be done in the CSS files instead?

Reply:
As far as I understand no: https://dash.plotly.com/dash-ag-grid/column-sizing#column-flex

But I may not know what CSS can do :)